.github/workflows/prerelease.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v3
         with:
-          go-version: '1.15'
+          go-version: '>=1.18'
```

Comment:
I discovered this problem: https://github.com/snowplow/snowplow-golang-analytics-sdk/issues/32 when I hit it in another project.

Because we had only previously tested this project on a fixed version (and we didn't think to bump that), we never discovered it even though we did a release since.

Setting this to '>=1.18' means the tests will run on whatever the latest version is after 1.18. It doesn't guarantee we catch bugs but it reduces the likelihood of the same thing happening again.

Reply:
Makes sense. That's good to know!

Reply:
Chiming in here that you could also use a matrix of versions to test on a variety of go-version each time.  So for example one test runs on the fixed 1.18 and another on the latest available.  This gives you broader coverage if you want to support a number of different versions.

Reply:
@jbeemster we do in the test workflow! https://github.com/snowplow/snowplow-golang-analytics-sdk/blob/2a73b99a79f3874f343914d6a018468a7b3ff337/.github/workflows/test.yml#L15

However, that doesn't cover us if we forget to update the matrix if a new go version is out. So, I chose `>=` here, so the release workflow always just uses the latest version, as an extra safety net. :)